src/Extensions/Activities/Internal/EventSource/ActivityEventSource.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics.Tracing;
+using Microsoft.Omex.Extensions.Abstractions.EventSources;
+
+namespace Microsoft.Omex.Extensions.Activities
+{
+	// Renamed from Microsoft-OMEX-TimedScopes to avoid conflict with sources in other libraries
+	[EventSource(Name = "Microsoft-OMEX-TimedScopes-Ext")]
```

Comment:
>Microsoft-OMEX-TimedScopes-Ext [](start = 22, length = 30)

shouldn't we change the name not to have TimedScopes? Do we keep it for backward compatibility ?

Reply:
It's used in our log aggregator and changing it would require redeploying configs for everybody (plus maintaining both during update). So I  think we could leave it like this until switching to 1DS when it could be simply deleted.